playbooks/lets-encrypt.yml

```diff
@@ -26,6 +29,9 @@
         address. Make sure the fully qualified domain resolves to the correct IP
         address before proceeding.
 
+        If Streisand has created a new server for you, its IP address is
+        {{ ansible_play_hosts }}
```

Comment:
Does this need to be `ansible_play_hosts[0]` (matching the comment on line 14)?

Reply:
For some reason I thought this was broken. It isn't. Thanks!

Reply:
Apparently it blows up CI if I do that.